cmd/users.go

```diff
@@ -108,6 +108,58 @@ from the web app.`,
 	},
 }
 
+var cmdDeploymentLogin = &cobra.Command{
+	Use:   "login [user]",
+	Short: "Authenticate with the remote",
+	Long:  "Retreives an access token from the remote using your credentials.",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		remoteName := strings.Split(cmd.Parent().Parent().Use, " ")[0]
+		deployment, _, err := local.GetClient(remoteName, configFilePath, cmd)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		username := args[0]
+		fmt.Print("Password: ")
+		pwBytes, err := terminal.ReadPassword(int(syscall.Stdin))
+		fmt.Println()
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		resp, err := deployment.LogIn(username, string(pwBytes))
+		if err != nil {
+			log.Fatal(err)
```

Comment:
we might want slightly better feedback here, in case daemon is offline and whatnot

Reply:
Good call. I'll update this.